src/components/Home/Hero.tsx

```diff
@@ -0,0 +1,22 @@
+import Image from "next/image";
+import React from "react";
+import bikeWoman1 from "../../assets/images/bike-woman 1.png";
+
+const Hero = () => (
+    <div className="flex flex-col md:flex-row sm:flex-row">
```

Comment:
Can you add `justify-center items-center` to this className so it centers the content on the page? Looks good otherwise